src/main/java/mekanism/common/block/BlockBounding.java

```diff
@@ -81,14 +81,27 @@ public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World wo
 	}
 
 	@Override
-	public boolean removedByPlayer(IBlockState state, World world, BlockPos pos, EntityPlayer player, boolean willHarvest)
```

Comment:
This will cause bounding boxes being broken to not break their parent.

Reply:
Oops. Fixed with new ```removedMainBlock``` method.